src/data.tsx

```diff
@@ -0,0 +1 @@
+export const SEARCH_DEFAULT: string = 'searchString';

```

Comment:
It's better to keep file with global variables in separate folder, you can call it shared/helpers/utils/ and etc.

Reply:
Fixed in https://github.com/ylakhtin/rs-react-components/commit/34e6f043dbf8bf9104f08ab45ea9f6b9ef07e5a1